content/docs/intro/cloud-providers/consul/setup.md

```diff
@@ -0,0 +1,28 @@
+---
+title: HashiCorp Consul Setup
+meta_desc: This page provides an overview on how to configure the Pulumi Consul Provider.
+---
+
+The [Pulumi Consul provider]({{< relref "./" >}}) uses the Consul SDK to manage resources.
+
+> Pulumi relies on the Consul SDK to authenticate requests from your computer to HashiCorp Consul. Your credentials are never sent
+> to pulumi.com.
+
+### Configuring The Provider
+
+Once obtained, there are two ways to communicate your configuration tokens to Pulumi:
```

Comment:
Nit: This says "tokens" plural, but below only shows a single env variable / config value. Should this be singular or the word "tokens" removed from the sentence? Or should it show setting the token env variable / config value?

Reply:
Changed to `communicate your configuration parameters`